api/include/opentelemetry/baggage/baggage_context.h

```diff
@@ -27,7 +27,7 @@ inline nostd::shared_ptr<Baggage> GetBaggage(const context::Context &context) no
 }
 
 inline context::Context SetBaggage(context::Context &context,
-                                   nostd::shared_ptr<Baggage> baggage) noexcept
+                                   const nostd::shared_ptr<Baggage> &baggage) noexcept
```

Comment:
Providing details because this is not trivial, as justification, for all reviewers:

* ABI change (the prototype is different)
* on a helper API
* which is inlined anyway, expanded in the caller compile unit

Hence:

* there is **no** ABI change overall, as no API implemented by the SDK is touched

Approved, this is safe IMO.